stdlib/3/multiprocessing/managers.pyi

```diff
@@ -17,6 +17,12 @@ class Namespace: ...
 
 _Namespace = Namespace
 
+class BaseProxy: ...
+
+class ValueProxy(BaseProxy, Generic[_T]):
+    def get(self) -> _T: ...
+    def set(self, value: _T): ...
```

Comment:
```suggestion
    def set(self, value: _T) -> None: ...
```

Also in the previous patch you had `value: _T` here. Could you re-add this?

Reply:
Sure, sorry, I misinterpreted removing the default value for removing the `value` attribute altogether.  Hopefully fixed along with the return for set.